xml/art_admin_slemicro_selinux.xml

```diff
@@ -0,0 +1,353 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+                 type="text/xml"
+                 title="Profiling step"?>
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<article xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="article-selinux" xml:lang="en">
+ <title>&slema; security</title>
+ <section xml:id="sec-introduction-selinux">
+  <title>Introduction to SELinux</title>
+  <para>
+   SELinux was developed as an additional Linux security solution that uses the
+   security framework in the Linux kernel. The purpose was to allow for a more
+   granular security policy that goes beyond the standard Discretionary Access
+   Controls (DAC), the traditional file permissions of owner/group/world, and
+   read/write/execute.
+  </para>
+  <para>
+   SELinux uses labels attached to objects (for example, files, binaries, and
+   network sockets) and uses them to determine privilege boundaries, thereby
+   building up a level of confinement that can span more than a process, or
+   even the whole system.
+  </para>
+  <para>
+   SELinux was developed in accordance with some certification standards like
+   Common Criteria and FIPS 140.
+  </para>
+  <para>
+   SELinux default action is to deny any access that would violate rules
+   defined by the policy. Other feature of SELinux that increases security is
+   that processes run within their own domain and cannot access files of other
+   processes that run in different domains. A process cannot access another
```

Comment:
That's not strictly true, so I would rephrase this to something like "SELinux allows strict confinement of processes up to the point where they can't access the files of other processes on the same machine anymore"